GradleBindings.VisualStudio/source.extension.vsixmanifest

```diff
@@ -12,7 +12,13 @@
         <Tags>xamarin, gradle, maven, jcenter</Tags>
     </Metadata>
     <Installation InstalledByMsi="false">
-        <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[12.0]" />
```

Comment:
@RoboNET hey, thanks for the pull-request, but doesn't it brake the VS2013 support?